po/extra/meson.build

```diff
@@ -1,8 +1,8 @@
-i18n.gettext('extra',
-  args: [
-    '--directory=' + meson.source_root(),
-    '--from-code=UTF-8'
-  ],
-  preset: 'glib',
```

Comment:
Why remove this preset?

Reply:
I believe that the glib preset is only used to make the `-cTRANSLATORS` option work which is not needed for metadata translation